src/components/container/sticky-bar/sticky-bar.ts

```diff
@@ -0,0 +1,67 @@
+import { CSSResultGroup, html, LitElement, TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+import { setAttribute, toggleDatasetEntry } from '../../core/dom';
+import { AgnosticIntersectionObserver } from '../../core/observers';
+
+import style from './sticky-bar.scss?lit&inline';
+
+import '../container';
+/**
+ * A container that sticks to the bottom of the page if slotted into `sbb-container`.
+ *
+ * @slot - Use the unnamed slot to add content to the sticky bar.
+ */
+@customElement('sbb-sticky-bar')
+export class SbbStickyBarElement extends LitElement {
+  public static override styles: CSSResultGroup = style;
+
+  private _intersector: HTMLSpanElement;
+  private _observer = new AgnosticIntersectionObserver((entries) =>
+    this._toggleShadowVisibility(entries),
+  );
+
+  protected override firstUpdated(): void {
+    this._updateIntersectionObserver();
+  }
+
+  private _toggleShadowVisibility(entries: IntersectionObserverEntry[]): void {
+    entries.forEach((entry) => {
+      toggleDatasetEntry(this, 'settled', entry.isIntersecting || entry.boundingClientRect.top < 0);
+    });
+  }
+
+  private _updateIntersectionObserver(): void {
```

Comment:
The only place where this function is called is firstUpdated(), so do we really need the disconnect and reconnection? I feel that it could be necessary to call it in connectedcallback as well. Can you test removing, moving and re adding a sticky bar in DOM?